src/Raven.Server/Documents/Indexes/Persistence/Lucene/CustomIndexWriter.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Lucene.Net.Analysis;
+using Lucene.Net.Index;
+using Lucene.Net.Store;
+using Raven.Client.Documents.Indexes;
+
+namespace Raven.Server.Documents.Indexes.Persistence.Lucene
+{
+    public class CustomIndexWriter : IndexWriter
```

Comment:
Can we have a better name here? Maybe `TimeTrackingIndexWriter` ? Something that would indicate that that it updates the timings of the stats

Reply:
done